core/src/main/java/dk/alexandra/fresco/lib/generic/BroadcastComputation.java

```diff
@@ -17,13 +15,19 @@
     Computation<List<byte[]>, BuilderT> {
 
   private final List<byte[]> input;
+  private final boolean runValidation;
 
-  BroadcastComputation(List<byte[]> input) {
+  public BroadcastComputation(List<byte[]> input, boolean runValidation) {
     this.input = input;
+    this.runValidation = runValidation;
   }
 
-  BroadcastComputation(byte[] input) {
-    this(java.util.Collections.singletonList(input));
+  public BroadcastComputation(byte[] input, boolean runValidation) {
+    this(java.util.Collections.singletonList(input), runValidation);
```

Comment:
while you're touching this line might as well get rid of the long class name `java.util.Collections`